genome_designer/utils/__init__.py

```diff
@@ -17,6 +17,8 @@
 
 
 def are_fastas_same(fasta_1, fasta_2):
```

Comment:
If this function is only used in testing code, it should be located in testing_utils.py